end_to_end/gemma/7b/1_test_gemma.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# This file is both an integration test that runs once a day on a CPU machine with 257.77 GB RAM \
```

Comment:
Ok this is super confusing because (a) there are multiple files and (b) why 257.77GB of RAM... why that number?


```
This file, combined with step 2 in the same directory, demonstrates converting a Gemma checkpoint from Kaggle and running various MaxText operations on it.

This step is tested nightly on an ordinary CPUVM.
```